src/Session/commands.py

```diff
@@ -0,0 +1,48 @@
+class Commands:
+    def __init__(self):
+        pass
+
+    def add_food(self):
+        pass
+
+    def remove_food(self):
+        pass
+
+    def create_freezer(self):
+        pass
+
+    def create_fridge(self):
+        pass
+
+    def create_profile(self):
+        pass
+
+    def help(self):
+        pass
+
+    def list_food_storages(self):
+        pass
+
+    def list_fridges(self):
+        pass
+
+    def list_freezers(self):
+        pass
+
+    def load(self):
+        pass
+    
+    def open(self):
+        pass
+
+    def save(self):
+        pass
+
+    def delete_fridge(self):
+        pass
+
+    def delete_freezer(self):
+        pass
```

Comment:
I think we can just hate one method called `delete` and it can take in a parameter called `food_storage`. The delete methods were a really good call though!!

Reply:
Added this in as well!